app/views/steps/shared/_under_age_section.html.erb

```diff
@@ -1,4 +1,4 @@
-<p>
+<p class="govuk-body">
```

Comment:
The list in this partial need the new style classes. Also, as usual the `govuk-link` class in the links.